### ISSUE TITLE
This plugin doesn't seem to work anymore?

### ISSUE THREAD

**Issue by flayks** [pauloamgomes/Cockpit-MultipleCollectionLink#5]
I was looking for a solution to use multiple links on a Collection-Link as I would like to pick an element from any collection, but this plugin seems to be broken with the latest version of Cockpit?

I get this error in the console and it blocks the whole backoffice:
![image](https://user-images.githubusercontent.com/273716/80841175-2ca2a780-8bff-11ea-8cbe-473be6b3458d.png)

Maybe I'm doing something wrong?

**Comment by pauloamgomes** [comment]
Thanks for flagging, can you confirm the cockpit version you are using?

**Comment by flayks** [comment]
I cloned the next branch like a few days ago

**Comment by pauloamgomes** [comment]
It seems fine to me, can't replicate!

What's your field configuration? Does error disappear if you disable the MultipleCollectionLink addon? you can disable by adding in the config YAML:

```
modules.disabled:
 - MultipleCollectionLink
```

**Comment by flayks** [comment]
That is weird. My install is pretty standard without much in it. I disabled it in my `config.php` file like that:
```php
'modules.disabled' => ['Cockpit-MultipleCollectionLink'],
```
and the admin shows up again.

Tried as well on the cleanest install from the repo, same issue.
![image](https://user-images.githubusercontent.com/273716/80908849-a7fd7980-8d23-11ea-88aa-56afffa98480.png)

Could it come from my local env? (I'm using Valet on macOS)
